build/Dockerfile

```diff
@@ -1,6 +1,4 @@
-FROM quay.io/operator-framework/ansible-operator:v0.12.0
-
-RUN pip3 install --user jmespath
+FROM quay.io/operator-framework/ansible-operator:v0.14.1
```

Comment:
I've seen comments in chat about needing to use operator-sdk < 0.13 to build. Will this no longer be the case now that this is updated?

Reply:
This is changing the base ansible-operator image. Unrelated to the requirement to use `operator-sdk` binary of version 0.12.0.

(The reason we are locked to 0.12.0 of operator-sdk is because the `operator-sdk olm-catalog gen-csv` is broken for Ansible operators > 0.12.0)

Reply:
ACK - thanks for the explanation.